src/dmd/aggregate.d

```diff
@@ -839,8 +838,7 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
     }
 
     // Back end
-    Symbol* stag;   /// tag symbol for debug data
-    Symbol* sinit;  /// initializer symbol
+    void* sinit;  /// initializer symbol
```

Comment:
Nice! Those have always been bothering me. Any idea how we could efficiently tackle that without relying on `void*` ?

Reply:
Well the problem is that the result of `dmd.tocsym:toInitializer` as the code is written now, needs to be cached (without resulting in a much larger diff).

I mean we could use a global `Symbol*[Aggregate declaration]` but that feels just as dirty.